### ISSUE TITLE
Unsupported major.minor version 52.0

### ISSUE THREAD

**Issue by paleozogt** [layerhq/gradle-git-repo-plugin#13]
```
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.9.5
BuildVersion:	13F34

$  java -version
java version "1.7.0_45"
Java(TM) SE Runtime Environment (build 1.7.0_45-b18)
Java HotSpot(TM) 64-Bit Server VM (build 24.45-b08, mixed mode)

$ ./gradlew build
Download https://repo1.maven.org/maven2/com/layer/gradle-git-repo-plugin/2.0.0/gradle-git-repo-plugin-2.0.0.pom
Download https://repo1.maven.org/maven2/com/layer/gradle-git-repo-plugin/2.0.0/gradle-git-repo-plugin-2.0.0.jar

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/asimmons/Development/testproj/build.gradle' line: 17

* What went wrong:
A problem occurred evaluating root project 'TestProj'.
> java.lang.UnsupportedClassVersionError: com/layer/gradle/gitrepo/GitRepoPlugin : Unsupported major.minor version 52.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 4.828 secs
```

**Comment by paleozogt** [comment]
I think the jar is compiled against java 1.8.  Could you target 1.7?

**Comment by drapp** [comment]
I just published 2.0.1 based on this branch
https://github.com/layerhq/gradle-git-repo-plugin/pull/14
might take a bit to propagate. Let me know if that works
